paymentsheet/src/main/java/com/stripe/android/paymentsheet/paymentdatacollection/ach/USBankAccountFormViewModel.kt

```diff
@@ -348,8 +348,11 @@ internal class USBankAccountFormViewModel @Inject internal constructor(
     }
 
     fun onDestroy() {
+        _result.tryEmit(null)
         collectBankAccountLauncher?.unregister()
         collectBankAccountLauncher = null
+        reset()
+        onCleared()
```

Comment:
I think we can remove this, as it doesn’t do anything.

Reply:
Cool, thanks, I wasn't completely sure!